### DISCUSSION TITLE
Multicast group

### DISCUSSION THREAD

**Discussion by rahav** [contiki-ng/contiki-ng#1692] [Q&A]
I'm trying to adapt `examples\multicast` to use an application layer multicast group. In root.c I have the following:

    uip_ip6addr(&ipaddr, 0xFF13,0,0,0,0,0,0,0x48);
    mcast_conn = udp_new(&ipaddr, UIP_HTONS(MCAST_SINK_UDP_PORT), NULL);

In sink.c:

    uip_ip6addr(&addr, 0xFF13,0,0,0,0,0,0,0x48);
    rv = uip_ds6_maddr_add(&addr);

Messages are sent but not received. What am I doing wrong?

Attached is Cooja printout with my debug messages.
<img width="687" alt="Screen Shot 2021-11-12 at 16 50 23" src="https://user-images.githubusercontent.com/2912950/141579539-6ed457ff-538e-4106-93aa-55c05f74e12b.png">

**Comment by rahav** [comment]
@g-oikonomou I'm reaching out to you in desperation as this question has been open for a while and you are the author of the idea man of multicast. The question is conceptually simple: I would like to multicast only to a subset of nodes that join a multicast group. Is that possible in Contiki? What Contiki function does a joiner need to call?

**Reply by rahav** [reply]
@g-oikonomou An even more basic question is what multicast addresses (beyond ALL_MPL_FORWARDERS) is Contiki going for forward? Thank you in advance for your answers.

**Reply by g-oikonomou** [reply]
Linked with #1699
